apps/shell/messages/en.json

```diff
@@ -35,6 +35,7 @@
     "address-label": "Address",
     "copied": "Copied!",
     "click-to-copy": "Click to copy",
+    "click-to-copy-address": "Click to copy {{address}}",
```

Comment:
**In the[ common PR](https://github.com/haqq-network/frontend/pull/990)** I will go through all places where 'click-to-copy' key is used and refactor this to save a whole string .

**to have this**

```tsx
<Tooltip
  text={
    isEthAddressCopy
      ? t('copied', 'Copied!')
      : t('click-to-copy-address', 'Click to copy {address}', {
        address: ethAddress,
      })
  }
>
```

**instead of this**
```tsx
  <Tooltip
    text={
      isEthAddressCopy
        ? t('copied', 'Copied!')
        : `${t('click-to-copy', 'Click to copy')} ${ethAddress}`
    }
  >
```